CONTRIBUTING.md

```diff
@@ -690,3 +698,37 @@ When merging pull requests, follow these guidelines:
   It should be valid Markdown, be comprehensive, read like a changelog entry,
   and assume that the reader has no access to the diff.
 * Delete branches for merged PRs (by maintainers pushing to the main repo).
+
+### Marking PRs as "deferred"
+
+We sometimes use the ["deferred" label](https://github.com/python/typeshed/labels/deferred)
+to mark PRs and issues that we'd like to accept, but that are blocked by some
+external factor. Blockers can include:
+
+- An unambiguous bug in a type checker (i.e., a case where the
+  type checker is not implementing [the typing spec](https://typing.readthedocs.io/en/latest/spec/index.html)).
+- A dependency on a typing PEP that is still under consideration.
+- A pending change in a related project, such as stub-uploader.
```

Comment:
This reminds me of the [implementation-pending](https://github.com/python/typeshed/labels/implementation-pending) label. How should that interact with this policy and the "deferred" label?

Reply:
probably we don't really need both labels -- we could probably merge them into one :)

Reply:
We have too many labels anyway. They could use a good cleanup. But I agree that merging implementation-pending into deferred seems like an easy first step.

Reply:
the `project` and `project-discussion` labels could also probably be merged

Reply:
I guess the idea of "implementation-pending" is that we're specifically waiting for a third-party library we're stubbing to release their implementation. That's maybe a little different from "deferred", but not sure it's worth a separate label.